functions/_tide_item_nvm.fish

```diff
@@ -0,0 +1,11 @@
+function _tide_item_nvm
+    set -l node_version (nvm current 2> /dev/null)
+
+    if test -n "$node_version"
```

Comment:
The test command has the `-a` flag to combine expressions. I haven't verified it, but in general more commands = more time. Please combine these expressions with `-a`.

Reply:
Also completed and tested :)